release/mac/homebrew/sqlcmd.rb

```diff
@@ -0,0 +1,52 @@
+class Sqlcmd < Formula
+  desc "Microsoft SQL Server command-line interface"
+  homepage "https://github.com/microsoft/go-sqlcmd"
+  url "https://github.com/microsoft/go-sqlcmd.git",
+  tag:      "v0.8.1",

```

Comment:
>tag:      "v0.8.1", [](http://example.com/codeflow?start=2&length=19)

shouldn't we have a pipeline task that generates this file dynamically and puts the tag that triggered the task here?

Reply:
I'm not sure yet, this file is actually held in the homebrew repo, I am keeping a copy here so folks can see the full picture for release management.  I am currently waiting to see if this .rb gets accepted into the homebrew repo.

If it does get accepted into the homebrew repo, I presume this repo should include the automation to push updates automatically to the homebrew repo.